com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStreamDescriptor.cs

```diff
@@ -17,39 +15,31 @@ public string PathToAbc
                 return System.IO.Path.Combine(Application.streamingAssetsPath, pathToAbc);
 #endif
             }
-            set { pathToAbc = value; }
+            internal set { pathToAbc = value; }
         }
 
         [SerializeField]
-        private AlembicStreamSettings settings = new AlembicStreamSettings();
+        AlembicStreamSettings settings = new AlembicStreamSettings();
         public AlembicStreamSettings Settings
         {
-            get { return settings; }
-            set { settings = value; }
+            get => settings;
+            set => settings = value;
```

Comment:
Lol, why? You use the first notation everywhere and suddenly change to use `=>`? If it's different "module", I don't mind, otherwise, consistency would be best.